canonical-web-and-design/maas-ui
# fix(ui): fix deploy error when there are no releases

## Done

- Fix the selector when there are no ubuntu releases.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

This could be a bit tricky to QA, but I was using a fresh local development MAAS with no images loaded.
- Using a MAAS with no ubuntu releases go to the machine list.
- Tick a deployable machine and click "Deploy" in the take action menu.
- You should not get an error.

## Fixes

Fixes: #2418.
